tests/config/kafka_override.yaml

```diff
@@ -34,6 +34,7 @@ affinity:
             operator: In
             values:
             - amd64
+            - arm64
```

Comment:
I never understood the need for these affinity rules as they aren't doing anything- what if we remove them completely?

Reply:
Guess we can try removing them since we're not using multi-arch cluster. I just hope I'm not missing anything.